DictConfig __getattr__ access, e.g. `cfg.foo`, is now raising a AttributeError if the key "foo" does not exist
